src/wp-admin/user-edit.php

```diff
@@ -733,6 +733,7 @@
 
 	</table>
 
+		<?php if ( wp_is_application_passwords_available_for_user( $user_id ) || ! wp_is_application_passwords_supported() ) : ?>
```

Comment:
Well that didn't work. Reapplying the formatting:

```suggestion
<?php if ( wp_is_application_passwords_available_for_user( $user_id ) || ! wp_is_application_passwords_supported() ) : // phpcs:disable Generic.WhiteSpace.ScopeIndent ?>
```